### PR TITLE
new to statsig menu

### PR DESCRIPTION
**Pull request by ankit1khare**
re-organized the new to statsig section

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://66ff9ba20c21d7cc1d4deae2--cozy-fox-0defec.netlify.app

File: docs/client/concepts/parameter-stores.mdx [hunk 1]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
 Only available for Statsig Mobile Client SDKs
 :::
 
-Parameter Stores are a new way of organizing and managing parameters in your mobile app via the Statsig console.  They are now available for Statsig Android, iOS, and React Native sdks.
+### **Parameter Stores: Simplifying Your Mobile App Configuration Management**
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:10]: I updated the page just now - its for client sdks, not just mobile.  youll have to rebase

File: docs/client/concepts/parameter-stores.mdx [hunk 2]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
 
-## How should I use Parameter Stores?
-Here is a suggested workflow:
-1. Create a parameter store for you team or project.  Parameter Stores are an object to hold related parameters.
-2. As you are adding to your mobile app, consider which variables you'd like to be controlled on the server-side, rather than hard coded into your app. These can be things like:
+---
+
+### **What Exactly is a Parameter Store?**
+
+Parameter Stores let you manage the parameters driving features in your app (boolean flags, text resources, numerical settings) without hardcoding them into your code. This allows you to dynamically adjust app behavior by decoupling configuration from code.
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:18]: "hardcoding them into your code"

is redundant wording

Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:18]: "This allows you to dynamically adjust app behavior by decoupling configuration from code."

I dont see how

File: docs/client/concepts/parameter-stores.mdx [hunk 3]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+1. **Create a Boolean Parameter**
+
+   In the Statsig Console, create a boolean parameter called `enable_new_onboarding` and set its default value to `false`. This will be your feature toggle.
+
+2. **Use the Parameter in Your Code**
+
+   Fetch the value of the parameter in your app using the Statsig SDK:
+
+   ```java
+   boolean newOnboardingEnabled = StatsigClient.getConfig("enable_new_onboarding").getBoolean("value", false);
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:37]: its not getconfig, its getParameterStore("store_name").getBoolean("enable_new_onboarding")

File: docs/client/concepts/parameter-stores.mdx [hunk 4]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+   if (newOnboardingEnabled) showNewOnboardingFlow();
+   else showOldOnboardingFlow();
+   ```
+
+   The app reacts dynamically based on whether the new onboarding flow is enabled, without hardcoding the condition.
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:42]: and without tieing it to a specific gate or experiment

File: docs/client/concepts/parameter-stores.mdx [hunk 5]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+
+   Once you're ready to roll out the new onboarding flow to a specific user segment, remap the `enable_new_onboarding` parameter to a Feature Gate that targets users on the latest version.
+
+4. **Control Updates in Real-Time**
 
-- A boolean parameter to control access to a new feature.  Even though you may be used to using Feature Gates for boolean based feature management, start with a boolean parameter instead
-- A string parameter for a text resource that you may want to swap or experiment with
-- A number parameter that you use as input to an onboarding flow (onboarding_steps), a list length to truncate at, etc
-... and many more
+   When you're ready to release the new flow to more users, simply update the parameter in the Statsig Console—no need to release a new app version.
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:50]: This example still doesnt sell it- you could just use a feature gate in this example

File: docs/client/concepts/parameter-stores.mdx [hunk 6]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+
+---
+
+### **Best Practices for Using Parameter Stores**
+
+- **Start Small**: Begin with a simple, non-critical feature like `show_promo_banner` and create a boolean parameter to control it remotely.
+- **Experiment with Dynamic Configs**: Remap parameters to Dynamic Configs when you need real-time changes based on user behavior or app version.
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:57]: we dont really target based on user behavior?

File: docs/client/concepts/parameter-stores.mdx [hunk 7]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+
+### **Code Example: Feature Toggle**
+
+Here’s a quick example for Android:
+
+```java
+StatsigClient.initialize(context, "client-sdk-key", statsigUser, () -> {
+    boolean featureEnabled = StatsigClient.getFeatureGate("enable_new_feature").isEnabled();
+    if (featureEnabled) showNewFeature();
+    else showOldFeature();
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx]: the code is wrong

just remove this section. We will make a guided tutorial separately

File: docs/client/concepts/parameter-stores.mdx [hunk 8]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+Now that you understand how Parameter Stores work, here’s how you can take full advantage of them:
 
-Parameter Stores are available in:
-- Android SDK v4.33.0+
-- iOS SDK v1.45.0+
-- @statsig/js-client v1.4.0+
-- Dart SDK v1.2.0+
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:36]: This is really important, please leave this in the doc

File: docs/client/concepts/parameter-stores.mdx [hunk 9]
```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
...
+### **Next Steps for Maximizing Parameter Stores**
 
-## I'm not sold
-If you've had this problem already, you'll recognize the power immediately.  If you're just getting started using a Statsig SDK in your mobile app - start with Parameter Stores from the beginning.  Parameter Stores are inspired by [Facebook's Mobile Config](https://atscaleconference.com/mobile-configuration-at-meta-the-key-to-mobile-agile-development-at-scale/). [Uber's engineering blog](https://www.uber.com/blog/supercharging-a-b-testing-at-uber/) explains this same idea.  [Firebase Remote Config](https://firebase.google.com/docs/remote-config) works similarly.  The most serious companies in mobile use this to move faster, maintain backwards compatibility, and experiment more freely.  The extra configuration in the Statsig console is not as convenient as creating a gate, but it saves you in the future when your app version is shipped and does not hard code any Statsig entity values.  That's why we are only introducing Parameter Stores for mobile - your backend or website can usually be updated much more quickly, so you can swap out Statsig entities as you please.
```
Comment (tore-statsig) [inline docs/client/concepts/parameter-stores.mdx:30]: I'd like to keep a reference to how this is inspired by facebooks mobileconfig

**Pull request merged**